serge/configs/google_ts_config.serge

```diff
@@ -13,9 +13,8 @@ sync
         data
         {
             executable                          python
-            pull_command                        python /var/tms/translation_service/src/serge_ts_connector.py --mode pull_ts
-            push_command                        python /var/tms/translation_service/src/serge_ts_connector.py --mode push_ts --translation_api google --google_key_path /var/tms/secrets/GoogleKey.json
-
+            pull_command                        python /var/tms/translation_service/src/serge_ts_connector.py --mode pull_ts --serge_dir /var/tms/serge/ts --ts_serge_dir /var/tms-data/po_files/serge_copy --ts_inbox /var/tms-data/po_files/inbox --ts_outbox /var/tms-data/po_files/outbox
+            push_command                        python /var/tms/translation_service/src/serge_ts_connector.py --mode push_ts --serge_dir /var/tms/serge/ts --ts_serge_dir /var/tms-data/po_files/serge_copy --ts_inbox /var/tms-data/po_files/inbox --ts_outbox /var/tms-data/po_files/outbox --translation_api google --google_key_path /var/tms/secrets/GoogleKey.json
```

Comment:
Can we update this naming not a blocker. Currently the template is named `/secrets/TrnslationGoogleKey.json.template` so I figured we can stay consistent
```
--google_key_path /var/tms/secrets/TranslationGoogleKey.json
```

Reply:
Yes! I'll fix that.